build-system/server.js

```diff
@@ -504,11 +504,15 @@ function replaceUrls(mode, file) {
     file = file.replace('https://cdn.ampproject.org/a4a-v0.max.js', '/dist/amp-inabox.js');
     file = file.replace(/https:\/\/cdn.ampproject.org\/v0\//g, '/dist/v0/');
     file = file.replace('https://cdn1.ampproject.org/viewer/google/v5.js', 'https://cdn.ampproject.org/viewer/google/v5.js');
+    file = file.replace(/https:\/\/3p.ampproject.net\/(?:\d+\/)?(.+)\.html/,
```

Comment:
If this is only for local dev purpose, you can just point the src to localhost directly?